s4e-web/cypress/integration/auth/register.spec.ts

```diff
@@ -0,0 +1,44 @@
+/// <reference types = "Cypress" />
+
+import { Registration } from '../../page-objects/auth/register.po'
+
+context("Register", () => {
+	beforeEach(() => {
```

Comment:
this same as in login

- validation of form test cases into context
- this `beforeEach` to `before`
- correct registration should be placed after the context